src/hooks/Tokens.ts

```diff
@@ -31,7 +31,16 @@ function useTokensFromMap(tokenMap: TokenAddressMap, includeUserAdded: boolean):
 
     // reduce to just tokens
     const mapWithoutUrls = Object.keys(tokenMap[chainId]).reduce<{ [address: string]: Token }>((newMap, address) => {
-      newMap[address] = tokenMap[chainId][address].token
+      let token = tokenMap[chainId][address].token
+
+      if (token.equals(WETH[chainId]))
```

Comment:
why does tokenMap does not have the correct form of weth?

Reply:
Because the list is wrong in this case. Ideally this should be changed on the token list but we don't control that.